src/main/java/ch/njol/skript/classes/data/BukkitEventValues.java

```diff
@@ -37,6 +37,7 @@
 import com.destroystokyo.paper.event.block.AnvilDamagedEvent;
 import com.destroystokyo.paper.event.entity.ProjectileCollideEvent;
 import com.destroystokyo.paper.event.player.PlayerArmorChangeEvent;
+import com.destroystokyo.paper.event.entity.EntityJumpEvent;
```

Comment:
Can you remove this one specific lin? Just so that no changes to this file appear 😅

Reply:
Oh, I didn't notice! Thanks for reminding me.

:D